values/validator.yml

```diff
@@ -0,0 +1,46 @@
+---
+quorumFlags:
+  privacy: false
+  removeKeysOnDelete: false
+  isBootnode: false # Besu only, set this to true if this node is a bootnode
+  usesBootnodes: false # Besu only, set this to true if the network you are connecting to use a bootnode/s that are deployed in the cluster
+
+cluster:
+  provider: local # choose from: local | aws | azure
+  cloudNativeServices: false
+  reclaimPolicy: Delete # set to either Retain or Delete; note that PVCs and PVs will still exist after a 'helm delete'. Setting to Retain will keep volumes even if PVCs/PVs are deleted in kubernetes. Setting to Delete will remove volumes from EC2 EBS when PVC is deleted
+
+aws:
+  # the aws cli commands uses the name 'quorum-sa' so only change this if you altered the name
+  serviceAccountName: quorum-sa
+  # the region you are deploying to
+  region: ap-southeast-2
+
+azure:
+  serviceAccountName: quorum-sa
+  # the clientId of the user assigned managed identity created in the template
+  identityClientId: azure-clientId
+  keyvaultName: azure-keyvault
+  # the tenant ID of the key vault
+  tenantId: azure-tenantId
+  # the subscription ID to use - this needs to be set explictly when using multi tenancy
+  subscriptionId: azure-subscriptionId
+
```

Comment:
What is this AWS and azure stuff...

Reply:
https://github.com/hyperledger/firefly-helm-charts/pull/76/commits/55610c410186a352fe848b61fb5761986a4f0309